go/src/meguca/cache/frontends.go

```diff
@@ -209,7 +218,9 @@ var BoardPageFE = FrontEnd{
 	},
 
 	RenderHTML: func(data interface{}, json []byte) []byte {
-		s := templates.IndexThreads(data.(PageStore).Data.Threads, json)
+		// ModLog is empty here???
+		board := data.(PageStore).Data
+		s := templates.IndexThreads(board.Threads, json, board.ModLog)
```

Comment:
Could you tell me what calls this specific `RenderHTML`?
I already know ModLog is in the cache, so if there's some sort of generation I can probably hook it into that.

Reply:
Is [this](https://github.com/bakape/meguca/pull/851/files/3c302c0690f71234742a884e77c3e6c0fd1235c2#diff-740f8f053967cc460226420c3ae174dbR15) not in the cache?